lib/openapi_parser/schema_validators/string_validator.rb

```diff
@@ -86,5 +89,17 @@ def validate_uuid_format(value, schema)
 
         return [nil, OpenAPIParser::InvalidUUIDFormat.new(value, schema.object_reference)]
       end
+
+      def validate_date_format(value, schema)
+        return [value, nil] unless schema.format == 'date'
+
+        begin
+          Date.iso8601(value)
```

Comment:
```suggestion
          Date.strptime(value, "%Y-%m-%d")
```


Thank you for your PR!

`date` is defined in OAS 3.0 so this PR is very good 👍 
However, in the OAS definition, date is ` full-date ` and does not accept ` time-offset`.
But `iso8601` method accepts time-offset.
`date` and `date-time` is not same definition in OAS 3.0 so it is better not to accept ` time-offset ` in `date`.

So please use `Date#strptime` to reject `time-offset`.

Reply:
Fixed! thanks.